extensions-contrib/tdigestsketch/src/main/java/org/apache/druid/query/aggregation/tdigestsketch/TDigestSketchUtils.java

```diff
@@ -80,14 +79,14 @@ static int getMaxIntermdiateTDigestSize(int compression)
     return intermediateSize + 1000;
   }
 
-  static void throwExceptionForWrongType(ColumnValueSelector selector)
+  static void throwExceptionForWrongType(Object obj)
   {
-    final String msg = selector.getObject() == null
+    final String msg = obj == null
```

Comment:
nit: Oops, sorry I missed this earlier, I guess this null can't happen anymore so this logic could be simplified.